### PR TITLE
Celine

### PR DESCRIPTION
**Pull request by Sigmatecture**
To be reviewed by Instruct

### CODE REVIEW

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -1082,6 +1082,37 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		target: "normal",
 		type: "Ice",
 	},
+	// Celine
+	statusguard: {
+		accuracy: 100,
+		basePower: 0,
+		category: "Status",
+		desc: "Protects from physical moves. If hit by physical move, opponent is either badly poisoned, burned, or paralyzed at random and is forced out. Special attacks and status moves go through this protect.",
+		shortDesc: "If opponent uses physical attack, then gets status and phazed. User is protected.",
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]: This shortDesc is too long. A good way to test is by doing `/dt tackle` and clicking inspect on the description of the move and replacing it with your description. If it gets cut off, it's too long

Comment (KrisXV) [reply data/mods/ssb/moves.ts]: also it's phrased kinda weird

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -1082,6 +1082,37 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+		name: "Status Guard",
+		pp: 10,
+		priority: 4,
+		flags: {protect: 1},
+		onTryMovePriority: 100,  // don't understand this and next line
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]: these are required to make animations perform properly and fail correctly for the custom moves

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -1082,6 +1082,37 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+			this.add('-anim', source, 'Protect', source);
+		},
+		onModifyMove(move, pokemon) {
+			move.secondaries = [];
+			const statuses = ['brn', 'par', 'tox'];
+			move.secondaries.push({
+				chance: 100,
+				status: this.sample(statuses),
+			});
+		},
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]: this doesnt seem like the correct way to implement this. I would look at King's Shield's implementation for reference

File: data/mods/ssb/abilities.ts [hunk 4]
```diff
@@ -279,6 +279,18 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
+	// Celine
+	guardianarmor: {
+		shortDesc: "Raises Defense and Special Defense by two stages upon switch in.",
+		isNonstandard: "Custom",
+		onModifyDef(def) {
+			return this.chainModify(2);
+		},
+		onModifySpD(spd) {
+			return this.chainModify(2);
+		},
```
Comment (ghost) [inline data/mods/ssb/abilities.ts]: I think we should go the route of the Dauntless shield, here

Comment (KrisXV) [inline data/mods/ssb/abilities.ts]: This is just a direct stat modifier (like Eviolite or Fur Coat) so I would look at Drampa's abilities for a better idea on how to do this

**Review by KrisXV — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Also, it is **extremely** important to work on the ssb4 branch on github, NOT master branch!

File: data/mods/ssb/moves.ts (hunk 1 above)
Comment (Sigmatecture) [reply data/mods/ssb/moves.ts]: Protects from damaging attacks. Contact: burn, toxic, or para.

**Comment by Sigmatecture** [comment]
Thanks for the comments, I'll implement those and change to the ssb4 branch.

**Pull request closed without merging**